test/lib/utils/config/definition.js

```diff
@@ -21,6 +21,8 @@ t.test('basic definition', async t => {
     default: 'some default value',
     defaultDescription: '"some default value"',
     type: [Number, String],
+    hint: '<key>',
+    usage: '--key <key>|--key <key>',
```

Comment:
We can address this once we add a config item that has this kind of type to a command's params.